db/migrate/20170602132735_fix_group_allow_membership_requests.rb

```diff
@@ -0,0 +1,13 @@
+class FixGroupAllowMembershipRequests < ActiveRecord::Migration
+  def up
+    execute <<~SQL
+    UPDATE groups g
+    SET allow_membership_requests = 'f'
+    WHERE COALESCE((SELECT 1 FROM group_users gu WHERE gu.owner = 't' AND gu.group_id = g.id LIMIT 1), 0) != 1
```

Comment:
Why not use `WHERE NOT EXISTS(SELECT 1 FROM group_users gu WHERE gu.owner = 't' AND gu.group_id = g.id)`? Seems simpler.